packages/core/src/utils/ailaModeration/moderationPrompt.ts

```diff
@@ -1,46 +1,87 @@
 import moderationCategories from "./moderationCategories.json";
 
-export const moderationPrompt = `
-OBJECTIVE:
-
-You are a content moderation supervisor examining a lesson plan document which has been generated by a user through interactions with Oak National Academy's lesson planning assistant. Your job is to assess and moderate the lesson plan across a variety of categories.
-
-For each category group, you must determine the degree to which the lesson plan is compliant with the following guidelines, providing a single rating on a 1-5 Likert scale that encompasses the following categories.
-
-CATEGORY GROUPS:
-
-${moderationCategories
-  .map(
-    (categoryGroup) => `<category-group>
-'${categoryGroup.title}' contains the following categories:${categoryGroup.categories
-      .map(
-        (category) => `
-- ${category.title}: ${category.llmDescription}`,
-      )
-      .join("")}
-
-${
-  categoryGroup.note
-    ? `Note:
-${categoryGroup.note}
-`
-    : ""
-}
+type ModerationCategory = {
+  title: string;
+  abbreviation: string;
+  code: string;
+  llmDescription: string;
+  criteria5: string;
+  criteria4?: string;
+  criteria3?: string;
+  criteria2?: string;
+  criteria1: string;
+};
+
+export function generateNewModerationPrompt(
+  categories: ModerationCategory[],
+): string {
+  const categoryDetailsText = categories
+    .map(
+      (cat) => `
+<category>
+Category Title: '${cat.title}'
+Output Score Code (Abbreviation): '${cat.abbreviation}'
+Original Full Code (for context): '${cat.code}'
+Description for Assessment: ${cat.llmDescription}
 Rating Criteria:
-- 5 ${categoryGroup.criteria5}
-- 1 ${categoryGroup.criteria1}
-</category-group>
+- 5: ${cat.criteria5}${
```

Comment:
Could be a `ratingCriteriaText` function with a `list.filter().map().join()` which might be a bit nicer than all the ternaries?